quickstart/docker/minimal/compose.yml

```diff
@@ -0,0 +1,50 @@
+services:
+  initialize:
+    image: busybox
+    command: chown -Rc ${ZIGGY_UID:-1000} /persistent
+    user: root
+    environment:
+      HOME: /persistent
+      # PFXLOG_NO_JSON: "true"
+    volumes:
+    - ./persistent:/persistent
+  quickstart:
+    depends_on:
+      initialize:
+        condition: service_completed_successfully
+    image: ${ZITI_QUICK_IMAGE:-docker.io/openziti/ziti-cli}:${ZITI_QUICK_TAG:-latest}
+    restart: unless-stopped
+    build:
+      context: ${ZITI_SRC_ROOT:-../../../} 
```

Comment:
what's all this for? as an example, this is confusing

Reply:
Added the following explanation to [the applicable section of the README](https://github.com/openziti/ziti/blob/minimal-docker-quickstart/quickstart/docker/minimal/README.md?plain=1#L47).

> By adding this `--build` option to the `up` command, the container image is built from the Dockerfile with your locally built `ziti` binary instead of pulling the default `openziti/ziti-cli` container image from Docker Hub. In the `compose.yml`, the Docker build context is defined with environment variable `ZITI_SRC_ROOT` which defaults to `../../../` (three levels up from this directory at the top level of a Git working copy of the source repo).